kits/nl-to-sql-agent/README.md

```
@@ -0,0 +1,383 @@
+
+[![Live Demo](https://img.shields.io/badge/🚀_Live_Demo-Try_Queryline-brightgreen?style=for-the-badge)](https://agent-kit-omega.vercel.app/login)
+[![Built with Lamatic](https://img.shields.io/badge/Built_with-Lamatic_AgentKit-6366F1?style=for-the-badge)](https://lamatic.ai)
+
+<br/>
+
+[![Next.js](https://img.shields.io/badge/Next.js-14.2.15-black?style=flat-square&logo=next.js)](https://nextjs.org)
+[![React](https://img.shields.io/badge/React-18.3.1-61DAFB?style=flat-square&logo=react)](https://react.dev)
+[![TypeScript](https://img.shields.io/badge/TypeScript-5-3178C6?style=flat-square&logo=typescript)](https://www.typescriptlang.org)
+[![Microsoft SQL Server](https://img.shields.io/badge/Database-MSSQL-CC2927?style=flat-square&logo=microsoftsqlserver)](https://www.microsoft.com/en-us/sql-server)
+[![Tailwind CSS](https://img.shields.io/badge/Tailwind_CSS-3.4-38B2AC?style=flat-square&logo=tailwindcss)](https://tailwindcss.com)
+[![Read-only](https://img.shields.io/badge/Queries-Read--Only_%F0%9F%94%92-blue?style=flat-square)]()
```

markdownlint-cli2 (0.23.2)

[warning] No empty links

(MD042, no-empty-links)